jacktrip.pro

```diff
@@ -22,10 +22,17 @@ equals(QT_EDITION, "OpenSource") {
 
 nogui {
   DEFINES += NO_GUI
-  QT -= gui
```

Comment:
We had a bug where our nogui builds were linked to qtgui. Can you check that we don't regress here?

Reply:
Yep. Will double check.

Reply:
Added this line back with no issues